limitedstore/limitedstore.go

```diff
@@ -82,8 +96,19 @@ func (store *LimitedStore) ensureSpace(size int64) error {
 		// Enough space is available to store the new upload
 		return nil
 	}
-
-	for id, _ := range store.uploads {
+  sorted_uploads := make(Pairlist, len(store.uploads))
+  i := 0
+  for u,h := range store.uploads {
+    sorted_uploads[i] = Pair{u, h}
+    i++
+  }
+  sort.Sort(sorted_uploads)
```

Comment:
Is there a particular reason why you used sort.Sort and then iterate through the slice manually instead of using sort.Reverse (https://golang.org/pkg/sort/#Reverse)? As far as I can tell from looking at the source code, they are equivalent in terms of efficiency.